.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull Request
+
+on:
+  pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Push Docker Image to Github Registry
+        uses: whoan/docker-build-with-cache-action@v5
+        with:
+          username: "${{ github.actor }}"
+          password: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
the name of the variable is something differnt. This seems wrong?

Reply:
Not sure what do you mean by that :) ? what do you refer to by "variable" and different compared to what?